README.md

```diff
@@ -0,0 +1,85 @@
+| openvpn_clients                    | `[]`                                              | List of client names for which certificates should be generated                                                                                                 |
+| openvpn_bridge_dev                 | `br0`                                             | Name of the bridge interface beeing used for bridgeing                                                                                                          |
+| openvpn_bridge                     |                                                   | Bridge settings to be configured for defaults see below                                                                                                         |
+| openvpn_dns_servers                | `["8.8.8.8","8.8.4.4"]`                           | List of DNS servers to push to the client                                                                                                                       |
+
+### LDAP object
+
+| Variable            | Default / Mandatory                       | Description                                                                                    |
+|---------------------|-------------------------------------------|------------------------------------------------------------------------------------------------|
+| url                 | `ldap://host.example.com`                 | Address of you LDAP backend with syntax ldap[s]://host[:port]                                  |
+| anonymous_bind      | `False`                                   | This is not an Ansible boolean but a string that will be pushed into the   configuration file, |
+| bind_dn             | `uid=Manager,ou=People,dc=example,dc=com` | Bind DN used if "anonymous_bind" set to "False"                                                |
+| bind_password       | `mysecretpassword`                        | Password of the bind_dn user                                                                   |
+| tls_enable          | `no`                                      | Force TLS encryption. Not necessary with ldaps addresses                                       |
+| tls_ca_cert_file    | `/etc/openvpn/auth/ca.pem`                | Path to the CA ldap backend. This must must has been pushed before                             |
+| base_dn             | `ou=People,dc=example,dc=com`             | Base DN where the backend will look for valid user                                             |
+| search_filter       | `(&(uid=%u)(accountStatus=active))`       | Filter the ldap search                                                                         |
+| require_group       |                                           | This is not an Ansible boolean but a string that will be pushed into the   configuration file, |
```

Comment:
Instead, user `when: group_base_dn is defined`

Reply:
fixed